### PR TITLE
chore/add-react-fs-structure

### PR DESCRIPTION
**Pull request by dgillen27**

### CODE REVIEW

**Comment by will-t-harris** [comment]
Thanks for getting this up @dgillen27! ✨ 

Some thoughts: 
- I think it would be helpful to update the `README` for the app in this directory explaining where stuff goes a bit.
- We had discussed several options for where to locate tests in this pattern -- had we landed on having different `tests` directories? My personal preference is to colocate test files next to the file they're testing (e.g. `utils.test.ts` living right next to `utils.ts`). I'm not sure how to effectively communicate that with an empty file structure though. 🤔

**Review by rmutter — APPROVED** [review]

**Review by pfuscoe — APPROVED** [review]
Thanks Dan! A couple thoughts on this:

- Re Will's comment on tests, I think we had settled on leaving this up to the developer but encouraging locating them close to the file - maybe we could add a note in the README on it
- Should we also add a components directory in `shared`?
- For `static/utils` - I might be forgetting if we talked about this, but what kinds of things would go here?

**Comment by dgillen27** [comment]
@pfuscoe 
> * Re Will's comment on tests, I think we had settled on leaving this up to the developer but encouraging locating them close to the file - maybe we could add a note in the README on it
- I think it would make sense to just include this in the README as well 👍 
> * Should we also add a components directory in `shared`?
- Yes definitely
> * For `static/utils` - I might be forgetting if we talked about this, but what kinds of things would go here?
- I think this might have been a mistake since we have a shared/utils.ts file

**Comment by will-t-harris** [comment]
@dgillen27 agreed that this would make sense in the `README` file

File: docs/react-fs-structure.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The `Static` folder is intended for static files such as:
+
+- Fonts
+- Images
+- SVGS
+- Videos
+
+## Tests
+
+While we currently don't have an established pattern for how we would like to write tests, we are currently suggesting that you colocate test files next to the file they're testing (e.g. utils.test.ts living right next to utils.ts).
```
Comment (will-t-harris) [inline docs/react-fs-structure.md:38]:
```suggestion
While we currently don't have an established pattern for how we would like to write tests, our current suggestion is to colocate test files next to the file they're testing (e.g. utils.test.ts living right next to utils.ts).
```

**Review by will-t-harris — APPROVED** [review]
Left some minor edits, but otherwise looks good to me. Thanks for pulling this together Dan! 🌈

**Review by pfuscoe — APPROVED** [review]
The new doc looks great to me. Thanks for this Dan!

**Pull request merged**